### PR TITLE
Adding requirments on impact & disruption post data format

### PR DESCRIPTION
**Pull request by tchevily**
# Description

This PR adds more validation on data format of impact and disruption  APIs

## Issue (optional)

Issue link: BOT-536

## Pull Request type (optional)

This is a bug fix

## How to test

Here are the following steps to test this pull request:

- It should be impossible to create an disruption without impact
- It should be impossible to create an impact without pt object

### CODE REVIEW

**Comment by vpassama** [comment]
I'm OK with this.

We can't/don't create empty disruptions and this is the same for impacts/pt_objects, each time we must have at least 1 element.

**Comment by tchevily** [comment]
Hi @vpassama,
thanks for giving us your opinion.
Best regards.

**Comment by pthegner** [comment]
Do not forget the swagger doc update

File: chaos/formats.py [hunk 1]
```diff
@@ -336,7 +336,7 @@ def get_object_format(object_type):
         'send_notifications': {'type': 'boolean'},
         'notification_date': {'type': ['string', 'null'], 'pattern': datetime_pattern}
     },
-    'required': ['severity']
+    'required': ['severity', 'objects']
```
Comment (pthegner) [inline chaos/formats.py]: Add `'minItems': 1` on objects

File: chaos/formats.py [hunk 2]
```diff
@@ -380,5 +380,5 @@ def get_object_format(object_type):
             'items': associate_disruption_property_input_format
         }
     },
-    'required': ['reference', 'cause', 'contributor']
+    'required': ['reference', 'cause', 'contributor', 'impacts']
```
Comment (pthegner) [inline chaos/formats.py]: Add `'minItems': 1` on impacts

**Pull request merged**